src/assets/translations/en-US.json

```diff
@@ -2414,6 +2414,7 @@
             "Navigation": "Navigation",
             "Notifications": "Notifications",
             "Organization": "Organization",
+            "Payment Waiver": "Payment Waiver",
```

Comment:
Please add the other translations for the rest of the languages supported

Reply:
done.